tests/shell/pipeline/commands/internal.rs

```diff
@@ -31,7 +31,8 @@ fn takes_rows_of_nu_value_strings_and_pipes_it_to_stdin_of_external() {
             "#
         ));
 
-        assert_eq!(actual.out, "AndKitKat");
+        // chop will remove the last escaped double quote from \"Estados Unidos\"
+        assert_eq!(actual.out, "\"Estados Unidos");
```

Comment:
`Estados Unido` should be the expected output 🙂

Reply:
I thought it would too, but due to the argument having spaces, it is adding surrounding `"` and counting them as characters in the argument.

I'll double check this.

Reply:
Ohhh, interesting. If quotes are being added, we should look into fixing that 😬

Reply:
Here is the difference in output between my Windows machine and macbook when logging the line from stdin inside `chop`

### Windows: 
```
Ok("Canada")
Ok("\"New Zealand\"")
Ok("Ecuador")
Ok("\"Estados Unidos\"")
```

### Mac: 
```
Ok("Canada")
Ok("New Zealand")
Ok("Ecuador")
Ok("Estados Unidos")
```

Reply:
Oh wow, that's really interesting. There's nothing in our code or the csv crate that would separate windows behaviour from other systems, so my guess is that the external `echo` on windows does this when there are spaces in an argument.

What happens if you drop the `^echo $it` line? There's a possibility that there may no longer be lines (I think `^echo` was adding newlines for us), so you may have to tweak the test. The main thing here is that we feed an internal command into the stdin of an external (which we get from `get origin | nu --testbin chop`).